CHANGELOG.md

```diff
@@ -1,3 +1,50 @@
+# 1.7.0-alpha03 (August 2024)
+
+_Changes since 1.7.0-alpha02_
+
+## Features
+### Multiple Platforms
+- [Skia is updated to m126](https://github.com/JetBrains/compose-multiplatform-core/pull/1486)
+- [Commonized `org.jetbrains.compose.material3:material3-window-size-class` module](https://github.com/JetBrains/compose-multiplatform-core/pull/1466)
+- [Commonized `org.jetbrains.compose.material3.adaptive:adaptive-*` modules](https://github.com/JetBrains/compose-multiplatform-core/pull/1468)
```

Comment:
```suggestion
- [Commonized `org.jetbrains.compose.material3.adaptive:adaptive*` modules](https://github.com/JetBrains/compose-multiplatform-core/pull/1468)
```

Should the dash be there? In [the PR](https://github.com/JetBrains/compose-multiplatform-core/pull/1468/files#diff-6fe33dbc3032d362b36542545eb0a488f590fb05af7f9cb10e1865313c63e15e) the `material3.adaptive:adaptive` module seems to also be included.

Reply:
thx, I've mistaken here